src/components/ImportRecord/ImportRecord.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import ky from 'ky';
+import { stripesConnect, withOkapiKy, CalloutContext } from '@folio/stripes/core';
+import { LoadingView } from '@folio/stripes/components';
+
+
+class ImportRecord extends React.Component {
+  static propTypes = {
+    id: PropTypes.string,
+    stripes: PropTypes.shape({
+      logger: PropTypes.object.isRequired,
+    }).isRequired,
+    resources: PropTypes.shape({
+      query: PropTypes.shape({
+        xid: PropTypes.string,
+      }).isRequired,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      query: PropTypes.shape({
+        update: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+    okapiKy: PropTypes.func.isRequired,
+    intl: PropTypes.shape({
+      formatMessage: PropTypes.func.isRequired,
+    }).isRequired,
+  };
+
+  static contextType = CalloutContext;
+
+  static manifest = Object.freeze({
+    query: {},
+  });
+
+  constructor(props) {
+    super(props);
+    this.loading = false;
+  }
+
+  componentDidUpdate() {
+    if (!this.loading) {
+      const xid = this.props.resources.query?.xid;
+      this.loading = true;
+      this.loadExternalRecord(xid);
+    }
+  }
+
+  loadExternalRecord = (xid) => {
+    const { id, intl } = this.props;
+
+    this.props.okapiKy('copycat/imports', {
+      timeout: 30000,
+      method: 'POST',
+      json: {
+        externalIdentifier: xid,
+        internalIdentifier: id,
+        profileId: 'f26df83c-aa25-40b6-876e-96852c3d4fd4' // XXX hardwired to UUID in mod-copycat reference data
```

Comment:
Could you please extract assign it to a constant somewhere above in this file. Something like

````js
const PROFILE_ID = 'f26df83c-aa25-40b6-876e-96852c3d4fd4';
````

should work.

Reply:
Addressed in fbcc28f149c20a238ef97830129537faab119fa3